### PR TITLE
Features L-N: set Baseline statuses

### PR DESCRIPTION
**Pull request by ddbeck**
## motion-path

| Key                                                                                                                  | Baseline | Low since date | Versions                                                                                                                    |
| :------------------------------------------------------------------------------------------------------------------- | :------: | :------------- | --------------------------------------------------------------------------------------------------------------------------- |
| **motion-path**                                                                                                      |  🆕 Low  | 2023-08-21     | Chrome 116<br>Chrome Android 116<br>Edge 116 🔑💎<br>Firefox 72<br>Firefox for Android 79<br>Safari 16<br>Safari on iOS 16  |
| [`css.properties.offset-anchor`](https://developer.mozilla.org/docs/Web/CSS/offset-anchor#browser_compatibility)     |  🆕 Low  | 2023-08-21     | Chrome 116<br>Chrome Android 116<br>Edge 116 🔑💎<br>Firefox 72<br>Firefox for Android 79<br>Safari 16<br>Safari on iOS 16  |
| [`css.properties.offset-distance`](https://developer.mozilla.org/docs/Web/CSS/offset-distance#browser_compatibility) |  🆕 Low  | 2022-09-12     | Chrome 55<br>Chrome Android 55<br>Edge 79<br>Firefox 72<br>Firefox for Android 79<br>Safari 16 🔑💎<br>Safari on iOS 16     |
| [`css.properties.offset-path`](https://developer.mozilla.org/docs/Web/CSS/offset-path#browser_compatibility)         |  🆕 Low  | 2022-03-15     | Chrome 55<br>Chrome Android 55<br>Edge 79<br>Firefox 72<br>Firefox for Android 79<br>Safari 15.4 🔑💎<br>Safari on iOS 15.4 |
| `css.properties.offset-path.path`                                                                                    |  🆕 Low  | 2022-09-12     | Chrome 64<br>Chrome Android 64<br>Edge 79<br>Firefox 72<br>Firefox for Android 79<br>Safari 16 🔑💎<br>Safari on iOS 16     |
| [`css.properties.offset-rotate`](https://developer.mozilla.org/docs/Web/CSS/offset-rotate#browser_compatibility)     |  🆕 Low  | 2022-09-12     | Chrome 56<br>Chrome Android 56<br>Edge 79<br>Firefox 72<br>Firefox for Android 79<br>Safari 16 🔑💎<br>Safari on iOS 16     |
| [`css.properties.offset`](https://developer.mozilla.org/docs/Web/CSS/offset#browser_compatibility)                   |  🆕 Low  | 2022-09-12     | Chrome 55<br>Chrome Android 55<br>Edge 79<br>Firefox 72<br>Firefox for Android 79<br>Safari 16 🔑💎<br>Safari on iOS 16     |

🔑💎 marks a release that contributes to the Baseline low date.<br>

## nesting

| Key                                                                                                                          |    Baseline    | Low since date | Versions                                                                                                                         |
| :--------------------------------------------------------------------------------------------------------------------------- | :------------: | :------------- | -------------------------------------------------------------------------------------------------------------------------------- |
| **nesting**                                                                                                                  | ❌ Not Baseline |                | Chrome ❌<br>Chrome Android ❌<br>Edge ❌<br>Firefox 117<br>Firefox for Android 117<br>Safari ❌<br>Safari on iOS ❌                  |
| [`css.selectors.nesting`](https://developer.mozilla.org/docs/Web/CSS/Nesting_selectors#browser_compatibility)                | ❌ Not Baseline |                | Chrome ❌<br>Chrome Android ❌<br>Edge ❌<br>Firefox 117<br>Firefox for Android 117<br>Safari ❌<br>Safari on iOS ❌                  |
| [`api.CSSStyleRule.cssRules`](https://developer.mozilla.org/docs/Web/API/CSSGroupingRule/cssRules#browser_compatibility)     |     🆕 Low     | 2023-08-29     | Chrome 112<br>Chrome Android 112<br>Edge 112<br>Firefox 117 🔑💎<br>Firefox for Android 117<br>Safari 16.5<br>Safari on iOS 16.5 |
| [`api.CSSStyleRule.deleteRule`](https://developer.mozilla.org/docs/Web/API/CSSGroupingRule/deleteRule#browser_compatibility) |     🆕 Low     | 2023-08-29     | Chrome 112<br>Chrome Android 112<br>Edge 112<br>Firefox 117 🔑💎<br>Firefox for Android 117<br>Safari 16.5<br>Safari on iOS 16.5 |
| [`api.CSSStyleRule.insertRule`](https://developer.mozilla.org/docs/Web/API/CSSGroupingRule/insertRule#browser_compatibility) |     🆕 Low     | 2023-08-29     | Chrome 112<br>Chrome Android 112<br>Edge 112<br>Firefox 117 🔑💎<br>Firefox for Android 117<br>Safari 16.5<br>Safari on iOS 16.5 |

🔑💎 marks a release that contributes to the Baseline low date.<br>

### CODE REVIEW

File: feature-group-definitions/line-clamp.yml [hunk 1]
```diff
@@ -1,5 +1,16 @@
 spec: https://drafts.csswg.org/css-overflow-3/#line-clamp
 caniuse: css-line-clamp
+status:
+  baseline: high
```
Comment (foolip) [inline feature-group-definitions/line-clamp.yml:4]: Since this requires a prefix, but `line-clamp` will probably also be shipped at some point, I think this one requires a note. Split into a separate PR?

File: feature-group-definitions/media-capture.yml [hunk 2]
```diff
@@ -1,5 +1,11 @@
 spec: https://w3c.github.io/html-media-capture/
 caniuse: html-media-capture
+status:
+  baseline: false
```
Comment (foolip) [inline feature-group-definitions/media-capture.yml:4]: Let's split this into a separate PR to figure out https://github.com/web-platform-dx/web-features/pull/461#issuecomment-1839195349

Comment (ddbeck) [reply feature-group-definitions/media-capture.yml:4]: Moved to https://github.com/web-platform-dx/web-features/pull/500

File: feature-group-definitions/line-clamp.yml (hunk 1 above)
Comment (ddbeck) [reply feature-group-definitions/line-clamp.yml:4]: Moved to https://github.com/web-platform-dx/web-features/pull/503.

**Comment by ddbeck** [comment]
Given that the rest of the features have received no comment, I am trusting that this is now OK to merge.
